compiler/src/main/java/com/squareup/anvil/compiler/ModuleMerger.kt

```diff
@@ -321,50 +327,17 @@ internal class ModuleMerger(
   }
 }
 
-@Suppress("DataClassPrivateConstructor")
-private data class AnnotationHolder private constructor(
```

Comment:
These wrapper classes weren't that useful anymore and it was easier to work with `AnnotationReference` directly instead.